### ISSUE TITLE
Results list bumped upwards when opening prepopulated search

### ISSUE THREAD

**Issue by ltGuillaume** [TBog/TBLauncher#120]
# Description #

When opening search with a prepopulated results list (with history items), the search bar and keyboard open, then the history shows, but then the quick list opens last. This makes the history items jump upwards. 

## Steps to Reproduce ##

To reproduce the behavior:
1. ✓ Use quick list
2. ✓ Desktop mode Search -> Initial result list -> anything but Empty
3. Open Search mode
4. See the build-up: first search bar, then results list, then result list jumping up and then quick list.

## Expected behavior ##

1. It would be best if search bar and quick list items were drawn before the results list is filled with the history items.
2. Also, I see an animation when the quick list icons appear, but not with the history items.

## Device info ##

  * OS: Android 10
  * TinyBit Launcher version: 6dev3

**Comment by TBog** [comment]
If you disable `Link keyboard with search bar visibility` does the issue still happen?
I ask this because I believe the keyboard is at fault, not the quick list. Even tho' the quick list has a slow animation, its location on screen is reserved from the beginning.

**Comment by ltGuillaume** [comment]
Nope, it doesn't!

**Comment by ltGuillaume** [comment]
Yes it's pretty hard to follow with the naked eye, but you're right, it is indeed that the results list is shown so quickly that the keyboard still needs to claim its space, after which the results list is pushed upwards.

**Comment by TBog** [comment]
I'll find a way to show the result list after the keyboard.

**Comment by ltGuillaume** [comment]
Great! Thx 😃

**Comment by TBog** [comment]
I added a delay that I hope will fix this glitch. See commit 9d29035f6568d908a7c6d8f55b271badd6e7d238

**Comment by ltGuillaume** [comment]
Looks all right to me, I'll test it as soon as there's a new dev build.

Do the items also present with the transition effect that is also used when opening tags?

**Comment by TBog** [comment]
The latest release is [here](https://github.com/TBog/TBLauncher/releases/tag/v6.0-dev.4)

> Do the items also present with the transition effect that is also used when opening tags?

I'm not sure what you mean.

**Comment by ltGuillaume** [comment]
Well the quick list items seem to have a "vertical rotation" transition upon show, and when switching between tags, there also seems to be that same transition in the results list. It's _very_ fast and doesn't seem to be linked to Android's developers options (so I can't set it to slow down for testing), so it's not really clear, but I'm quite sure I don't see this when the results list is prepopulated.

As for the delay, that works just great over here!

**Comment by ltGuillaume** [comment]
So no animation? 😉

**Comment by TBog** [comment]
There is a fade in animation when the result list first appears. When the content changes the items get reordered, scaled in or just disappear without animation.
I still don't understand what animation you desire. Maybe open an issue with this feature request.

**Comment by ltGuillaume** [comment]
https://github.com/TBog/TBLauncher/issues/125
